articles/connections/passwordless/native-passwordless-universal.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Passwordless Authentication in Native Applications with Universal Login
+---
+# Passwordless Authentication in Native Applications with Universal Login
+
+[Universal Login](/hosted-pages/login) is the only recommended way to set up passwordless authentication in your native application. We recommend using the Auth0 login page for the best experience, security and the fullest array of features.
+
+Once you are done with this guide, authentication should occur in this fashion:
+
+1. A user clicks to login and the Auth0 SDK redirects them to the login page on the web.
+1. At this point, Lock Passwordless will ask them for a phone number or email, whichever type you chose
+1. The prompt will change to ask them for a code which they will receive by the designated method.
+1. Once they enter the code, the transaction will finish and the user will be redirected to your app along with their credentials.
+
+Implementing passwordless in the login page involves just a few steps.
+
+## 1. Set up your connection
+
+To set up the passwordless connection, go to your [Dashboard](${manage_url}/#/connections/passwordless) under **Connections > Passwordless** and turn on either Email or SMS. Then, go to the **Apps** tab and enable the connection for the clients that will be using it.
+
+## 2. Configure your login page
+
+Next, configure your login page. Head over to the [Hosted Pages](${manage_url}/#/login_page) section of your Dashboard. At the top of your the code editor, click the menu entitled **Default Templates** and pick **Lock (Passwordless)** from the list.
+
+## 3. Implement Auth0 in your application
+
+All that remains is to set up your application to call the login page. An easy to follow guide for this process already exists for both Swift and Android. Using Auth0's quickstart guides, follow the **Login** step through. It will guide you on how to call the login page from your application. The process for invoking universal login from a native app is the same whether `lock-passwordless` will be used inside the login page or not.
```

Comment:
All these sentences basically repeat the same thing `All that remains is to set up your application to call the login page. An easy to follow guide for this process already exists for both Swift and Android. Using Auth0's quickstart guides, follow the **Login** step through. It will guide you on how to call the login page from your application.` -> `Follow this tutorial to call the login page from your app`

Reply:
I can simplify this paragraph some, but not quite that much. IMO it's important that we note that we have guides only for those two technologies, and that they only need to follow the "login" step in particular, which shows them how to call the login page, and that the process is the same for invoking universal login regardless of what is used inside it (because the guide is technically for a different use case)